src/main/telemetry/ibus_shared.c

```diff
@@ -141,8 +141,12 @@ static uint8_t dispatchMeasurementRequest(ibusAddress_t address) {
     }
 #endif
     if (SENSOR_ADDRESS_TYPE_LOOKUP[address].value == IBUS_MEAS_VALUE_TEMPERATURE) { //BARO_TEMP\GYRO_TEMP
-        if (sensors(SENSOR_BARO)) return sendIbusMeasurement2(address, (uint16_t) ((DEGREES_TO_CENTIDEGREES(getCurrentTemperature()) + 50) / 10  + IBUS_TEMPERATURE_OFFSET)); //int32_t
-        else {
+        /*if (sensors(SENSOR_BARO)) return sendIbusMeasurement2(address, (uint16_t) ((DEGREES_TO_CENTIDEGREES(getCurrentTemperature()) + 50) / 10  + IBUS_TEMPERATURE_OFFSET)); //int32_t*/
```

Comment:
If I can suggest, it can be:
```
int16_t temperature;
if (sensors(SENSOR_BARO)) {
    getBaroTemperature(&temperature);
} else {
    getMPUTemperature(&temperature);
}
return sendIbusMeasurement2(address, (uint16_t) ((temperature + 50) / 10  + IBUS_TEMPERATURE_OFFSET));

```

Reply:
Why not, done, it now reports baro temp if available, IMU temp otherwise